### PR TITLE
remove the null byte check for now, add unicode test

### PR DESCRIPTION
**Pull request by CamDavidsonPilon**
I'm reverting a earlier PR that checks for null bytes, because it was failing with unicode characters.

### CODE REVIEW

File: pyreferrer/referrer_test.py [hunk 1]
```diff
@@ -425,49 +425,49 @@ def test_provided_twitter_url_overrides_what_is_present_in_twitter_useragent():
...
+        'url': '',
+        'subdomain': '',
+        'domain': '',
+        'label': '',
+        'tld': '',
+        'path': '',
+        'query': '',
+        'google_search_type': 'Not Google Search'
+    }
+    assert_equals(expected_referrer, referrer)
```
Comment (CamDavidsonPilon) [inline pyreferrer/referrer_test.py]: whitespace stuff

File: pyreferrer/referrer_test.py [hunk 2]
```diff
@@ -425,49 +425,49 @@ def test_provided_twitter_url_overrides_what_is_present_in_twitter_useragent():
...
-            'url': '1foo',
-            'subdomain': '',
-            'domain': '1foo',
-            'label': '1Foo',
-            'tld': '',
-            'path': '1foo',
-            'query': '',
-            'google_search_type': 'Not Google Search'
-        }
-        assert_equals(expected_referrer, referrer)
```
Comment (CamDavidsonPilon) [inline pyreferrer/referrer_test.py]: remove this test

File: pyreferrer/referrer_test.py [hunk 3]
```diff
@@ -425,49 +425,49 @@ def test_provided_twitter_url_overrides_what_is_present_in_twitter_useragent():
...
+        'query': u'', 
+        'tld': u'рф', 
+        'url': u'http://президент.рф/', 
+        'path': u'/', 
+        'subdomain': u'', 
+        'type': u'indirect', 
+        'google_search_type': u'Not Google Search', 
+        'label': u'Президент'
+    }
+    assert_equals(referrer, expected_referrer)
```
Comment (CamDavidsonPilon) [inline pyreferrer/referrer_test.py]: new test for unicode urls

**Comment by damnMeddlingKid** [comment]
👍

**Pull request merged**